src/integrationTest/java/org/opensearch/security/http/JwtAuthenticationWithUrlParamTests.java

```diff
@@ -93,6 +106,11 @@ public void shouldAuthenticateWithJwtTokenInUrl_positive() {
             response.assertStatusCode(200);
             String username = response.getTextFromJsonBody(POINTER_USERNAME);
             assertThat(username, equalTo(ADMIN_USER.getName()));
+            Map<String, String> expectedParams = Map.of("token", "REDACTED");
+
+            auditLogsRule.assertExactlyOne(
+                userAuthenticated(ADMIN_USER).withRestRequest(GET, "/_opendistro/_security/authinfo").withRestParams(expectedParams)
```

Comment:
I feel there should there be additional tests with other Params than token as REDACTED. Thoughts?

Reply:
I added another URL param here to ensure it stays unredacted. One of the issues in testing is that the URL param must be consumed by the API or else it returns an error like:

```
{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"request [/_plugins/_security/authinfo] contains unrecognized parameter: [notexist]"}],"type":"illegal_argument_exception","reason":"request [/_plugins/_security/authinfo] contains unrecognized parameter: [notexist]"},"status":400}
```

FYI this PR also fixes a bug in the way that the `verbose` param was consumed.